.github/workflows/disk-clean.yml

```
@@ -0,0 +1,36 @@
+name: Clean up disk space
```

```suggestion
name: disk-clean
```

We normally use the same as the filename.